hadoop-hdds/docs/content/concept/StorageContainerManager.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Storage Container Manager"
+date: "2017-09-14"
+weight: 3
+menu: 
+  main:
+     parent: Architecture
+summary:  Storage Container Manager or SCM is the core metadata service of Ozone. SCM provides a distributed block layer for Ozone.
+---
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+Storage Container Manager (SCM) is the leader node of the *block space management*. The main responsibility is to create and manage [containers]({{<ref "concept/Containers.md">}}) which is the main replication unit of Ozone.
+
+
+![Storage Container Manager](StorageContainerManager.png)
+
+## Main responsibilities
+
+Storage container manager provides multiple critical functions for the Ozone
+cluster.  SCM acts as the cluster manager, Certificate authority, Block
+manager and the Replica manager.
+
+SCM is in charge of creating an Ozone cluster. When an SCM is booted up via `init` command, SCM creates the cluster identity and root certificates needed for the SCM certificate authority. SCM manages the life cycle of a data node in the cluster.
+
+ 1. SCM is the block manager. SCM
+allocates blocks and assigns them to data nodes. Clients
+read and write these blocks directly.
+
+ 2. SCM keeps track of all the block
+replicas. If there is a loss of data node or a disk, SCM
+detects it and instructs data nodes make copies of the
+missing blocks to ensure high availability.
+
+ 3. **SCM's Ceritificate authority** is in
+charge of issuing identity certificates for each and every
+service in the cluster. This certificate infrastructure makes
+it easy to enable mTLS at network layer and the block
+token infrastructure depends on this certificate infrastructure.
+
+## Main components
+
+For a detailed view of Storage Container Manager this section gives a quick overview about the provided network services and the stored persisted data.
+
+**Network services provided by Storage Container Manager:**
```

Comment:
Sure, will create multiple new issues.

What do you think about adding a warning to the zh pages when the English page is changed?

```
WARNING: Information may not be up to date: the English version of this page is modified since the last translation. 
```

If you like this approach, can you please translate this sentence (or something similar?).

Reply:
I'm OK with that. Here is the draft message in Chinese. 

注意：本页面翻译的信息可能滞后，最新的信息请参看英文版的相关页面。

Reply:
Thanks a lot. I added the warning where the original page is updated and created jira issues for missing translations (for the new pages: (HDDS-4079,.....)